packages/affiliates/test/mocha-local/contracts.js

```diff
@@ -0,0 +1,109 @@
+const hre = require("hardhat");
```

Comment:
is this a new set of tests?

Reply:
nvm I see it was moved from `/truffle/` to this `/mocha-local/`. That's a bit painful that github keeps failing to detect these renaims.

Reply:
Yeah :/